docs/source/getstarted.rst

```diff
@@ -103,31 +111,39 @@ Next Steps - "Night-Night"
     b. For Windows systems, type the following and press **Enter**::
 
         copy 01_hello_world.py nightnight.py
-    
+
+Now, nightnight.py is saved in the same folder as 01_hello_world.py.
+	
 2. Open this new document in a source code editor or plain-text editor. Free source code editors, such as `PyCharm Community Edition <https://www.jetbrains.com/pycharm/>`_ , `Atom <https://atom.io>`_ , `Sublime <https://www.sublimetext.com>`_ , or `TextWrangler <http://www.barebones.com/products/textwrangler/>`_ can be found online. Anki does not provide tech support for third-party source code editors.
 
-3. Each line in the program relates to a specific function. To learn more, see :doc:`the Beginner's Tutorial </tutorial-beginner>`.
+3. Each line in the program relates to a specific function.
 
-4. Move to the line that starts with "robot.say_text"
+    a. ``import cozmo`` allows your program to access the Cozmo SDK code contained within the ``cozmo`` module.
+    b. Text sandwiched between three ``'`` marks is a Docstring. Docstrings are like comments, and are placed inside code to give information to the user.
+    c. ``robot.say_text("Hello World").wait_for_completed`` is the core of the program.
```

Comment:
In the browser, for some weird reason "is the core of the program" is bold. Can you spot what is going wrong?